### PR TITLE
creating keypress event

### PR DESCRIPTION
**Pull request by NataliaLKB**
for https://github.com/easter-egg-fun/shake/issues/3

cc @tsop14

Can you take a look and give me a :+1: if you are happy with the first part of the code

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+	document.onkeypress = function(e) {
+	    e = e || window.event;
+
+	    var charCode = (typeof e.which == "number") ? e.which : e.keyCode;
+
+	    if (charCode === 116 || charCode === 101 || charCode === 115 || charCode === 116) {
+	    	typedKeys.push(charCode);
+
+	    	if (areArraysEqual( typedKeys, keyCombo )) {
```
Comment (tsop14) [inline script.js]: 'arraysAreEqual' bit more readable.

**Comment by tsop14** [comment]
:+1:

File: script.js (hunk 1 above)
Comment (NataliaLKB) [inline script.js]: Makes sense! Will change now.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+(function () {
+	'use strict';
+
+	var keyCombo = [116, 101, 115, 116] // T E S T
```
Comment (tsop14) [inline script.js:4]: indentation :stuck_out_tongue_closed_eyes:

Comment (NataliaLKB) [inline script.js:4]: We don't have an editor config or anything, and because none of this code will go into production as it, does it matter for a hack?

Comment (NataliaLKB) [inline script.js:4]: Happy to add an editorconfig and or linter if you prefer

**Comment by tsop14** [comment]
I was being a :boar:

**Comment by NataliaLKB** [comment]
you are a :sunflower:

**Pull request merged**